app/params/config.go

```diff
@@ -93,11 +95,9 @@ func SetTendermintConfigs(config *tmcfg.Config) {
 	config.Consensus.GossipTransactionKeyOnly = true
 	config.Consensus.UnsafeProposeTimeoutOverride = 1 * time.Second
 	config.Consensus.UnsafeProposeTimeoutDeltaOverride = 500 * time.Millisecond
-
 	config.Consensus.UnsafeVoteTimeoutOverride = 50 * time.Millisecond
 	config.Consensus.UnsafeVoteTimeoutDeltaOverride = 500 * time.Millisecond
-	config.Consensus.UnsafeCommitTimeoutOverride = 50 * time.Millisecond
-
+	config.Consensus.UnsafeCommitTimeoutOverride = 100 * time.Millisecond
```

Comment:
can we keep this as 50ms? we can bump up if needed. This will add about 50ms to consensus if we make this change, and this is on top of waiting for 2/3 votes already